### PR TITLE
Add frontend instructions to DEVELOPMENT docs

### PR DESCRIPTION
**Pull request by AlanGreene**
https://github.com/tektoncd/dashboard/issues/51

Add instructions for running the frontend locally connected to any running backend.
Also include instructions for testing, linting, and storybook use.

### CODE REVIEW

**Comment by AlanGreene** [comment]
@steveodonovan can you take a look? This is mostly taken from the legacy frontend repo and updated to reflect the current state of tektoncd/dashboard. I've also updated the config.json to reference the same port as the backend docs. (1 less manual step for local setup)

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ You must install these tools:
 1. [`git`](https://help.github.com/articles/set-up-git/): For source control
 1. [`dep`](https://github.com/golang/dep): For managing external Go
    dependencies. - Please Install dep v0.5.0 or greater.
+1. [Node.js & npm](https://nodejs.org/): For building and running the frontend locally. See `engines` in [pacakge.json](./package.json) for versions used.
```
Comment (a-roberts) [inline DEVELOPMENT.md]: nit: package.json*

Comment (AlanGreene) [reply DEVELOPMENT.md]: oops, good catch

Comment (AlanGreene) [reply DEVELOPMENT.md]: fixed

**Comment by a-roberts** [comment]
/approve

Thanks @AlanGreene and @steveodonovan, this looks great, merging 😄

**Pull request merged**